### PR TITLE
Refactor Makefiles for core-ui and add-on-ui apps

### PR DESCRIPTION
**Pull request by kwiatekus**
**Description**

Changes proposed in this pull request:

- Adopt makefiles to comply with [generic prow jobs ](https://github.com/kyma-project/test-infra/pull/2317)

**Related issue(s)**
https://github.com/kyma-project/test-infra/pull/2317
https://github.com/kyma-project/console/issues/1744

### CODE REVIEW

File: stary makefile [hunk 1]
```diff
@@ -1,15 +1,16 @@
...
 .PHONY: resolve
 resolve:
 	npm run bootstrap:ci
 
 .PHONY: validate
 validate:
-	npm run conflict-check
-	npm run lint-check
-	npm run test-shared-lib
+	echo "badziewie"
```
Comment (parostatkiem) [inline stary makefile:13]: Are you sure it's "badziewie"?

Comment (kwiatekus) [reply stary makefile]: file removed

**Pull request merged**